rlclientlib/vw_model/vw_model.cc

```diff
@@ -162,6 +173,26 @@ int vw_model::request_multi_slot_decision(const char* event_id, const std::vecto
   }
 }
 
+const std::string vw_model::add_optional_audit_flag(const std::string& command_line) const
+{
+  if (_audit) { return command_line + " --audit"; }
+
+  return command_line;
+}
+
+void vw_model::write_audit_log(const char* event_id, const char* audit_buffer) const
+{
+  if (event_id != nullptr && audit_buffer != nullptr)
+  {
+    std::string filename = _audit_output_path + reinforcement_learning::delimiter + std::string(event_id);
```

Comment:
Do we need to ensure that this path does not reference some important system path and overwrite it?

Reply:
+1. Also, there is no limitation on entity_id characters - some of them can be problematic for paths ('/','\',...)

Reply:
The path itself can be define by the caller to it's up to them to use an appropriate path. I will add path sanitization.